commands/commands.go

```
@@ -67,20 +127,37 @@ func migrate(c *components.Context, migrateToRtfs bool) error {
 		return errors.New("Need to provide two arguments while provided " + strconv.Itoa(len(c.Arguments)))
 	}
 
+	log.Info("Starting migration process")
```

the go file is only a simple validation wrapper, not sure such a message here is correct. you could fail validation after you stated you begin the process (jar not found, configuration prepare fails etc)
further more, i see in file "migrationBase.java#startMigration" we already log that we are starting (and that's the real start)